frameworks/angular/creating-workspace.md

```diff
@@ -112,7 +112,7 @@ export class AppComponent {
    "licenseKey": "openfin-demo-license-key",
    "runtime": {
       "arguments": "--v=1 --inspect",
-      "version": "33.116.77.11"
+      "version": "38.126.83.74"
```

Comment:
This runtime will not work with v19 or v19.1 (right now until it is updated). The openfin/workspace-platform npm package restricts initialisation to supported runtimes so it should be the runtime listed in the workspace section of the openfin versions page.